### ISSUE TITLE
[BUG] import the review.assembly does not work

### ISSUE THREAD

**Issue by YaoZhou89** [theaidenlab/juicebox#1027]
**Describe the bug**
Our genome size is ~4Gb and the scale factor is 2. When I load the .hic file, the default scale factor is set to 3. Next, I imported the .assemlby file and set the scale factor to 2. Then, I continued to import the .review.assembly, the scale factor was changed to 3 again and the heatmap was not changed as expected. Although I set the scale factor to 2 again, heatmap is still the original one.

**To Reproduce**
1. Go to 'file -> open ' and load the local .hic file
2. Go to 'Assembly -> Import Map Assembly' and import the original .assembly file.
3. Go to 'Assembly -> Import Modified Assembly' and import the manually reviewed assembly file (.review.assembly)
4. Scroll down to 'Set Scale' and set the to 2
5. See error (the heatmap keep the same)

**Expected behavior**
The heatmap should change according to the .review.assembly file.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS Montery
 - Version 2.17.00

**Additional context**

**Comment by paka14** [comment]
Yes, I also met the same situation. Is it  a bug? How can I solve it?

**Comment by mankiddyman** [comment]
Experiencing this exact bug as well.. please let us know if you found a fix or workaround!

**Comment by kaskelso** [comment]
@YaoZhou89 Did you find a solution to this? I am currently dealing with the same exact issue.

Edit: For anyone else who ends up with this issue see https://github.com/c-zhou/yahs/issues/94

Basically, this is caused by the program producing the .hic file having a different scaling method from Juicebox. You need to have the scaling methods match. To do this you can use the modified juicer in HapHiC to regenerate the .hic file https://github.com/zengxiaofei/HapHiC/tree/main
